docs/guide.md

```
@@ -406,6 +427,33 @@ If you encounter issues during setup, here are common problems and solutions:
+
+1. **Confirm topic subscription**
+   ```sh
+   curl "http://localhost:9091/api/v1/topics?topic=<topic>&nodeinfo=true"
+   ```
+   Ensure the expected peers appear with `nodeinfo=true`.
+
+2. **Inspect mesh propagation**
+   ```sh
+   curl http://localhost:9091/api/v1/p2p-snapshot | jq
+   ```
+   
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)